client/admin-pointers.js

```diff
@@ -0,0 +1,26 @@
+/*global wcSevicesAdminPointers, ajaxurl */
+import jQuery from 'jquery';
+
+jQuery( document ).ready( function( $ ) {
+	const pointers = wcSevicesAdminPointers;
+
+	function show_pointer( i ) {
+		const pointer = pointers[ i ];
+		const options = $.extend( pointer.options, {
+			close: function() {
+				$.post( ajaxurl, {
+					pointer: pointer.id,
+					action: 'dismiss-wp-pointer',
+				} );
+				if ( pointers[ i + 1 ] ) {
+					show_pointer( i + 1 );
+				}
+			},
+		} );
+		$( pointer.target ).pointer( options ).pointer( 'open' );
+	}
+
+	if ( pointers[ 0 ] ) {
```

Comment:
Technically not, since if there are no notices to display, WordPress won't run the script. However, I agree about being defensive so I added this & other checks in f50a15e5db729b04978bd1ca532bd12b38bc0de7